lib/ui/compositing.dart

```diff
@@ -280,12 +280,14 @@ class SceneBuilder extends NativeFieldWrapperClass2 {
   }) {
     assert(_matrix4IsValid(matrix4));
     assert(_debugCheckCanBeUsedAsOldLayer(oldLayer, 'pushTransform'));
-    final TransformEngineLayer layer = TransformEngineLayer._(_pushTransform(matrix4));
+    final EngineLayer engineLayer = EngineLayer._();
+    _pushTransform(engineLayer, matrix4);
+    final TransformEngineLayer layer = TransformEngineLayer._(engineLayer);
     assert(_debugPushLayer(layer));
     return layer;
   }
 
-  EngineLayer _pushTransform(Float64List matrix4) native 'SceneBuilder_pushTransform';
+  void _pushTransform(EngineLayer layer, Float64List matrix4) native 'SceneBuilder_pushTransform';
```

Comment:
Can we adopt a convention to indicate that these are output parameters of the native method?  (e.g. prefixing these parameter names with `out_`?)

Reply:
How about just `out` iwthout the underscore?  Snake case looks weird in Dart.

But that sounds fine to me.

Reply:
`out` with no underscore SGTM